doc/nrf/libraries/caf/power_manager.rst

```diff
@@ -102,84 +123,78 @@ There are some events that reset the power-down counter:
 Suspended
 ---------
 
-Upon power-down timeout, the |power_manager| switches the application to the suspended state if there is at least one module that restricts power-down levels to :c:enum:`POWER_MANAGER_LEVEL_SUSPENDED` and there is no module that restricts power below level to any higher state (:c:enum:`POWER_MANAGER_LEVEL_ALIVE`).
-
-The other modules of the application, if applicable, will turn off the peripherals or switch them to standby to conserve power.
-The operating system is kept in the :c:enum:`PM_STATE_ACTIVE` state.
-
-It is assumed that the operating system will conserve power by setting the CPU state to idle whenever possible.
-The established connection is maintained.
+Upon the power-down timeout, the |power_manager| switches the application to the suspended state.
+The |power_manager| suspends application modules to reduce power consumption.
+After suspending the application modules, the |power_manager| might call :c:func:`sys_poweroff`, so that system enters the off state to reduce the power consumption further.
 
-Off
----
-
-Upon power-down timeout, the |power_manager| switches the application to the deep sleep mode if no module restricts it.
-This means that all the modules are restricted to :c:enum:`POWER_MANAGER_LEVEL_OFF` or :c:enum:`POWER_MANAGER_LEVEL_MAX`.
-
-If applicable, the other modules of the application turn off the peripherals or switch them to standby to conserve power.
-The operating system switches to the :c:enum:`POWER_STATE_DEEP_SLEEP_1` state.
-The devices are suspended and the CPU is switched to the deep sleep (off) mode.
-
-A device reboot is required to exit this state.
-
-Error
------
+.. _caf_power_manager_suspending_application_modules:
 
-The |power_manager| checks if any application modules have reported an error condition.
+Suspending application modules
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When any application module switches to the error state (that is, broadcasts :c:enum:`MODULE_STATE_ERROR` through :c:struct:`module_state_event`), the |power_manager| puts the application into the error state.
-Then, after the period of time defined by :kconfig:option:`CONFIG_CAF_POWER_MANAGER_ERROR_TIMEOUT`, it puts the application to the off state.
-During this period, other modules can report the error condition to the user (for example, :ref:`caf_leds` can keep working in the error state).
+The |power_manager| submits :c:struct:`power_down_event`.
+While handling the event, application modules turn off the used peripherals or switch them to standby to conserve power.
+Only after all application modules confirm that they have entered the low power state (by not consuming the :c:struct:`power_down_event`), the |power_manager| sets the required application power state.
+The |power_manager| registers itself as the final subscriber of the :c:struct:`power_down_event`.
+Reception of the :c:struct:`power_down_event` indicates that the |power_manager| can continue power down sequence.
```

Comment:
```suggestion
Only after all application modules confirm that they have entered the low power state (by not setting the :c:struct:`power_down_event`), the |power_manager| sets the required application power state.
The |power_manager| registers itself as the final subscriber of :c:struct:`power_down_event`.
Reception of :c:struct:`power_down_event` indicates that the |power_manager| can continue power down sequence.
```

Reply:
`consuming` is more proper here

```
The event handler gets a pointer to the app_event_header structure as the function argument. The function should return true to consume the event, which means that the event is not propagated to further listeners, or false, otherwise.
```
Ref: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/others/app_event_manager.html#implementing_an_event_handler_function